loggerInFile.js

```diff
@@ -1,16 +1,16 @@
 const winston = require('winston');
```

Comment:
it is better to rename this file back to `logger.js` and create `loggerEnv` here.

Reply:
rename back to logger,js.Dont know why,but  I cant create two loggers in one file becaouse they have some conflicts between each other.